node/node.go

```diff
@@ -808,7 +809,7 @@ func (n *Node) OpenDatabase(name string, cache, handles int, namespace string, r
 // also attaching a chain freezer to it that moves ancient chain data from the
 // database to immutable append-only files. If the node is an ephemeral one, a
 // memory database is returned.
-func (n *Node) OpenDatabaseWithFreezer(name string, cache, handles int, ancient string, namespace string, readonly bool) (ethdb.Database, error) {
+func (n *Node) OpenDatabaseWithFreezer(name string, cache int, handles int, ancient string, namespace string, readonly bool, dbOptions map[string]interface{}) (ethdb.Database, error) {
```

Comment:
Thanks for the background info. Instead of naming it `LevelDBConfig`, maybe we can name it `ExtraDBConfig`, which will store config info for both leveldb and pebbledb. This way, we can use this struct everywhere for both db types in the future. What do you think?

Reply:
That sounds good to me!

Reply:
@cffls Please see the most recent commit.